packages/help-center/src/components/help-center-survey/help-center-survey.scss

```diff
@@ -0,0 +1,63 @@
+@import "@automattic/typography/styles/variables";
+@import "@wordpress/base-styles/breakpoints";
+
+.help-center__survey {
+	background: #f6f7f7;
+	display: flex;
+	flex-direction: column;
+	gap: 24px;
+	border-radius: 2px;
+	border-left: 3px solid #000;
```

Comment:
Is it possible to use a variable instead of #000?
This also applies to the other colors used here 👍